src/GraphQL.Tests/Types/Collections/SchemaTypesTests.cs

```diff
@@ -84,7 +84,8 @@ public void throws_exception_when_multiple_type_instances_exists()
 
         // Test
         Should.Throw<InvalidOperationException>(() => schema.Initialize())
-            .Message.ShouldBe("A different instance of the GraphType 'ObjectGraphType' with the name 'MyObject' has already been registered within the schema. Please use the same instance for all references within the schema, or use GraphQLTypeReference to reference a type instantiated elsewhere.");
+            .Message.ShouldBe(@"A different instance of the GraphType 'ObjectGraphType' with the name 'MyObject' has already been registered within the schema. Please use the same instance for all references within the schema, or use GraphQLTypeReference to reference a type instantiated elsewhere.
+To view additional trace enable GlobalSwitches.TrackGraphTypeInitialization switch.");
```

Comment:
:+1:

Setting global switches in tests is failure-prone without correct setup.  If you want to do so, be sure to apply `[Collection("StaticTests")]` to the class that contains the tests, so they do not run parallel to other tests.  (This is how we fixed the intermittent culture failures.)

Reply:
I would keep switch disabled.